frontend/src/routes/Applications/ApplicationDetails/ApplicationDetails.tsx

```diff
@@ -74,6 +75,12 @@ export const ApplicationContext = createContext<{
 const namespaceString = ':namespace'
 const nameString = ':name'
 
+// in case where application is just created
+// because recoil state is throttled
+// the new application might yet be in recoil state
+// so if application not found
+const WAIT_FOR_APPLICATION_DELAY = 1000
```

Comment:
I would suggest defining and exporting the constant value here:
https://github.com/stolostron/console/blob/main/frontend/src/atoms.tsx#L489

Reply:
That's not what I meant. I was suggesting to make a constant for the 500 ms value, and to use the same one in your code. Does the value need to be higher than 500? If so, it could be that you always wait for twice the polling time in your code.

Reply:
done